src/components/ui/Chip.svelte

```diff
@@ -1,23 +1,32 @@
 <script lang="ts">
   import CloseIcon from '@nasa-jpl/stellar/icons/close.svg?component';
   import { createEventDispatcher } from 'svelte';
+  import { shadeColor } from '../../utilities/color';
   import { classNames } from '../../utilities/generic';
 
+  export let color: string = '#f8f8f8';
+  export let disabled: boolean = false;
   export let label: string = '';
+  export let className: string = '';
   export let removable: boolean = true;
+  export let role: string = 'button';
```

Comment:
Do you mind typing this as `AriaRole` if that makes sense? My mind is deep in the user role hole so my first thought was that this was for users and was a little confused for a sec. That's probably just me though lol.

Reply:
Oh good call, didn't know that was a type.

Reply:
Actually not sure that exists (at least not natively in TS). Could rename the prop to `ariaRole`?